src/main/java/com/example/DoughnutShop.java

```diff
@@ -0,0 +1,20 @@
+package com.example;
+
+public class DoughnutShop {
+
+    public String prepareDoughnut(String doughnutType) {
+        String preparedDoughnut;
+
+        if (doughnutType.equals("Chocolate")) {
```

Comment:
could we maybe create a `Doughnut` interface to represent these? we could then implement each flavour as its own class. this is quite a big structural change though so up to you, but i feel like it may make this more extensible!